### PR TITLE
Add bugsnag plugin to marketplace

### PR DESCRIPTION
**Pull request by Irma12**
Signed-off-by: irma12 <irma@roadie.io>

## Hey, I just made a Pull Request!

This PR adds our newly created Bugsnag plugin ( https://roadie.io/backstage/plugins/bugsnag/?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=bugsnag) to Backstage marketplace :) 

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/data/plugins/bugsnag.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Bugsnag
+author: roadie.io
+authorUrl: https://sda.se/
```
Comment (dhenneke) [inline microsite/data/plugins/bugsnag.yaml:4]: 👀

Comment (Irma12) [reply microsite/data/plugins/bugsnag.yaml]: @dhenneke Thank you for catching that :) Sorry for the confusion

**Comment by jhaals** [comment]
Small line ending change should get prettier to pass. You should be able to get it to format in VSCode or using `cd microsite && yarn prettier format -w .`

**Comment by Irma12** [comment]
> Small line ending change should get prettier to pass. You should be able to get it to format in VSCode or using `cd microsite && yarn prettier format -w .`

Thank you @jhaals  :)

**Review by jhaals — APPROVED** [review]

**Pull request merged**